/*
* Brian R Taylor
* brian.taylor@bolderflight.com
* 
* Copyright (c) 2020 Bolder Flight Systems
*/

/*
* This protobuf file describes the data log format, which is transpiled
* to a C header and source file for use in the flight code. There should
* only be a single message defined, DatalogMessage, which defines all
* of the data to be logged. The message structure should be flat
* (i.e. no nested messages), consisting of native types.
*/

syntax = "proto3";

import "nanopb.proto";

/* Data log message */
message DatalogMessage {
  repeated int32 inceptors = 1 [(nanopb).max_count = 16, (nanopb).fixed_count = true];
  float imu_accel_x_mps2 = 2;
  float imu_accel_y_mps2 = 3;
  float imu_accel_z_mps2 = 4;
}

